### DISCUSSION TITLE
Cant create Ventoy Stick. Unknown problem

### DISCUSSION THREAD

**Discussion by skorpion1298** [ventoy/Ventoy#1156] [General]
Hey there,

every time I try to use Ventoy on a 256GB Samsung Stick it fails.
The Errorlog is attached. I cant find a solution to this :/ Fails on every device I try to create
[log.txt](https://github.com/ventoy/Ventoy/files/7374362/log.txt)

I run Windows 10 64Bit.

**Comment by ventoy** [comment]
Please try:
1. open diskpart.exe and run `list disk` to find the physical drive number of the USB (in the log it's physical drive 5)
2. run following commands in diskpart.exe
```
select  disk  5
clean
```

3. After that run `exit` to exit diskpart.exe 
4. Retry Ventoy2Disk.exe to install

**Reply by skorpion1298** [reply]
This worked magically fine! Thank you! I used the default 1.0.55 release.

**Comment by ventoy** [comment]
Or you can try this CI release:
https://github.com/ventoy/Ventoy/actions/runs/1362894729

**Reply by skorpion1298** [reply]
Alright I´ll try! Thank you!
